ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -73,12 +74,18 @@ class HikariJdbcConnectionProvider(dataSource: HikariDataSource, healthPoller: T
     else
       Unhealthy
 
-  override def runSQL[T](block: Connection => T): T = {
+  override def runSQL[T](block: Connection => T, databaseMetrics: DatabaseMetrics): T = {
```

Comment:
Perhaps make this two parameter lists so the second can still use the `{ conn => … }` syntax?

```suggestion
  override def runSQL[T](databaseMetrics: DatabaseMetrics)(block: Connection => T): T = {
```